docs/source/index.md

```diff
@@ -1,21 +1,99 @@
-# Welcome to MALA's documentation!
+![image](./img/logos/mala_horizontal.png)
+
+# Welcome to MALA!
+
+## What is MALA?
+
+MALA is a software package for building ML models that replace
+density functional theory (DFT) calculations. DFT is **the** most
+popular method for simulating materials on a quantum level and predicting 
```

Comment:
DFT is one of the most widely used methods for simulating materials at a quantum level and predicting their properties, employed by researchers worldwide.